src/DOM/DOMRegistry.js

```diff
@@ -7,253 +7,254 @@ const xlink = 'http://www.w3.org/1999/xlink';
 const xml = 'http://www.w3.org/XML/1998/namespace';
 
 const DOMAttributeNamespaces = {
-    'xlink:actuate': xlink,
-    'xlink:arcrole': xlink,
-    'xlink:href': xlink,
-    'xlink:role': xlink,
-    'xlink:show': xlink,
-    'xlink:title': xlink,
-    'xlink:type': xlink,
-    'xml:base': xml,
-    'xml:lang': xml,
-    'xml:space': xml
+	'xlink:actuate': xlink,
+	'xlink:arcrole': xlink,
+	'xlink:href': xlink,
+	'xlink:role': xlink,
+	'xlink:show': xlink,
+	'xlink:title': xlink,
+	'xlink:type': xlink,
+	'xml:base': xml,
+	'xml:lang': xml,
+	'xml:space': xml
 };
 
 const DOMAttributeNames = {
-    acceptCharset: 'accept-charset',
-    className: 'class',
-    htmlFor: 'for',
-    httpEquiv: 'http-equiv',
-    xlinkActuate: 'xlink:actuate',
-    xlinkArcrole: 'xlink:arcrole',
-    xlinkHref: 'xlink:href',
-    xlinkRole: 'xlink:role',
-    xlinkShow: 'xlink:show',
-    xlinkTitle: 'xlink:title',
-    xlinkType: 'xlink:type',
-    xmlBase: 'xml:base',
-    xmlLang: 'xml:lang',
-    xmlSpace: 'xml:space',
-    viewBox: 'viewBox' // SVG - Edge case. The letter 'b' need to be uppercase
+	acceptCharset: 'accept-charset',
```

Comment:
It's not one space, it's one tab. I personally prefer spaces but trueadm prefers tabs, and it's his repo =)

The benefit of tabs is you can make them as wide as you want for readability. For example, to me the green line is showing as MORE space, not less. Screenshot attached.
![image](https://cloud.githubusercontent.com/assets/331032/11981877/7a06c668-a95b-11e5-937e-33330872726a.png)

Tabs usually show up as 4 spaces of 8 spaces by default, but most or all editors should be able to adjust it. If you want to have the whole repo switch to 2 or 4 spaces instead of 1 tab per indent, you'll have to convince the others.

Sorry for the confusion.